### PR TITLE
Edit's to Aaron's external contribution

### PR DESCRIPTION
**Pull request by DonNateR**
- Replaces PR #1595 

- Rename article.

- Make edits for clarity

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1c6e67190f/how-to/)

**Comment by DonNateR** [comment]
@aaron-medrano: Can you look over my edits and make sure everything checks out?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b879c5a903/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-56905b21bc/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2ee6f828a4/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-401ab9837f/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. My suggested edits and a few questions are in the attached Word doc (and PDF copy of it). Please let me know if you have any questions. Thanks!

[Set up Autodiscover for Outlook_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/609992/Set.up.Autodiscover.for.Outlook_editsKH.pdf)
[Set up Autodiscover for Outlook_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/609993/Set.up.Autodiscover.for.Outlook_editsKH.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-ef9fb38e0e/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-bb9715108a/how-to/). :bow:

File: content/exchange/set-up-autodiscover-for-outlook.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+created_date: '2016-10-25'
+created_by: Aaron Medrano
+last_modified_date: '2016-11-24'
+last_modified_by: Nate Archer
+product: Microsoft Exchange
+product_url: exchange
+---
+
+
+In order for certain features on your Exchange account to work with Outlook, you must set up your domain name with a special DNS record that points to Rackspace Cloud Office Autodiscover server.
```
Comment (kmsholcomb) [inline content/exchange/set-up-autodiscover-for-outlook.md]: ...points to the Rackspace ...

File: content/exchange/set-up-autodiscover-for-outlook.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- Free/busy information for calendar scheduling
+
+- Out of office setup and management through Outlook
+
+- Offline address book synchronization
+
+- Sharing calendars, contacts, and folders using a sharing invitation within Outlook
+
+- Access to Public Folders in Exchange 2013 and Exchange 2016.
```
Comment (kmsholcomb) [inline content/exchange/set-up-autodiscover-for-outlook.md]: Delete the period

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-423573ca2b/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a9dd9c4180/how-to/)

**Comment by kmsholcomb** [comment]
Looks good!

**Pull request merged**